andela/ah-backend-thor
# Chore fix heroku Deploy

### What does this PR do
Fixes the issue with Heroku deploy migrations error

#### Description of Task to be completed?
Ensure that Heroku link has all features merged in the develop branch.

#### How should this be manually tested?
visit https://ah-backend-thor.herokuapp.com/documentation/ 
and check out the documented features.

#### Any background context you want to provide?
Ensure that the database configuration corresponds with this below;

```
DATABASES = {
    'default': {
        'ENGINE': 'django.db.backends.postgresql_psycopg2',
        'NAME': 'ah_backend' or 'dcmdhglo8tdcjp',
        'USER': 'postgres' or 'bbnwhxoooyqlyt',
        'PASSWORD': '' or 'sudo' or 'kegz' or os.getenv("HEROKU_PASSWORD"),
        'HOST': 'localhost' or 'ec2-107-22-164-225.compute-1.amazonaws.com',
        'PORT': '5432',
    },
}
```

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/19260086/48209132-74879c00-e385-11e8-8eb1-4ec64a084c34.png)
